src/alembic/versions/003_fb033ef6de1b_add_reference_notes_column.py

```diff
@@ -0,0 +1,35 @@
+"""add reference_notes column
+
+Revision ID: fb033ef6de1b
+Revises: 0d3bf1348c58
+Create Date: 2024-07-09 09:03:11.190561
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "fb033ef6de1b"
+down_revision: Union[str, None] = "0d3bf1348c58"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
```

Comment:
opted for adding a new column instead of renaming the column, since I don't know how big these tables. and renaming/altering columns scares me.

Reply:
addressed with #15